app/views/ems_block_storage/show_list.html.haml

```diff
@@ -1 +1,7 @@
-#main_div= render :partial => "layouts/gtl"
+#main_div
+  - if ManageIQ::Providers::StorageManager.any?
+    = render :partial => 'layouts/gtl'
+  - else
+    = render :partial => 'layouts/empty',
+             :locals => {:add_message => _("Add a new Block Storage Manager"),
+                           :documentation => ::Settings.docs.cloud_provider}
```

Comment:
This should definitely not be `cloud_provider` here

Reply:
the links for documentations dont work for me (I tried to go to cloud docs, infra docs and got an error). I redirect it for now for  product_documentation_website and added comment that this should be set for a specific storage docs. who is responsible for writing the docs?

Reply:
fixed with @skateman help (see https://github.com/ManageIQ/manageiq/pull/20638)